homeassistant/components/litejet/light.py

```diff
@@ -1,43 +1,50 @@
 """Support for LiteJet lights."""
 import logging
 
-from homeassistant.components import litejet
 from homeassistant.components.light import (
     ATTR_BRIGHTNESS,
     SUPPORT_BRIGHTNESS,
     LightEntity,
 )
 
+from .const import DOMAIN
+
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_NUMBER = "number"
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
-    """Set up lights for the LiteJet platform."""
-    litejet_ = hass.data["litejet_system"]
+async def async_setup_entry(hass, config_entry, async_add_devices):
```

Comment:
```suggestion
async def async_setup_entry(hass, config_entry, async_add_entities):
```

We will be adding entities not devices